src/loconet/loconet_cv.c

```diff
@@ -190,7 +189,9 @@ uint8_t loconet_cv_set(uint16_t lncv_number, uint16_t lncv_value)
       // Set magic value to detect we have configured the address.
       page_data[1] = LOCONET_CV_DEVICE_CLASS;
       // Change lncv_address
-      lncv_address = lncv_value;
+      loconet_config.bit.ADDRESS = lncv_value;
```

Comment:
The problem here is that `loconet_cv_write_allowed` is now also used for the address and priority. See also #16